src/components/bar-chart/bar-chart.scss

```diff
@@ -0,0 +1,7 @@
+.bar-chart {
```

Comment:
please extract `bar chart` and `pie chart` into separate style-only  (BEM) component and reuse these styles in both charts -- and the remove these specific bar-chart and pie-chart files

Reply:
separate folder for one chart.scss style?